enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/ShiroAuthManager.java

```diff
@@ -98,16 +98,19 @@ public void start() throws Throwable
         users.start();
         roleRepository.start();
 
-        if ( authEnabled && realm.numberOfUsers() == 0 )
+        if ( realm.numberOfRoles() == 0 )
```

Comment:
We could have the authEnabled condition here as well to speed up the startup-time for severs with auth disabled